artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2447,6 +2447,14 @@
             </xsd:documentation>
          </xsd:annotation>
       </xsd:attribute>
+      <xsd:attribute name="sync" type="xsd:boolean" use="optional" default="true">
+         <xsd:annotation>
+            <xsd:documentation>
+               If this is true, producers will be waiting a response from the mirror before the sync is finished.
+               This is false by default.
```

Comment:
Seems conflicted:

default="true"

"This is false by default."

Reply:
you're right.. thanks

Reply:
Is there a simple test for this type of default that could be updated to check this value? I notice no test needed to change when you updated it, which probably explains how it came to be wrong without notice to start with.

Reply:
the default on parsing was actually correct... just the default value on the XSD was wrong.

Reply:
there was another test validating the default on broker properties.. and I just added a single assert on the next mirror option that wasn't touched on the XML

I think we're good

Reply:
I am marking it resolved.. please unresolved it if you still see anything else.